tutorials/multi_agent_gpu_training_with_warp_drive.ipynb

```diff
@@ -272,12 +322,11 @@
    "outputs": [],
    "source": [
     "env_registrar = EnvironmentRegistrar()\n",
-    "this_file_dir = os.path.dirname(os.path.abspath(\"__file__\"))\n",
     "env_registrar.add_cuda_env_src_path(\n",
     "    CovidAndEconomyEnvironment.name,\n",
     "    os.path.join(\n",
-    "        this_file_dir, \n",
-    "        \"../ai_economist/foundation/scenarios/covid19/covid19_build.cu\"\n",
+    "        _PATH_TO_AI_ECONOMIST, \n",
```

Comment:
ambiguous, path to a directory? to what?

Reply:
the path to the installed package folder